plugins/backfill/transformerExporter.go

```diff
@@ -13,7 +14,9 @@ type exporter string
 var Exporter exporter
 
 func (e exporter) Export() ([]event.TransformerInitializer, []storage.TransformerInitializer, []interface1.ContractTransformerInitializer) {
-	return []event.TransformerInitializer{},
+	return []event.TransformerInitializer{
+			cat_file_box.EventTransformerInitializer,
```

Comment:
I think this needs to be removed from this file if we don't want to kick off a backfill